src/Blocks/Expandable.purs

```diff
@@ -71,29 +94,44 @@ contentClasses status = contentSharedClasses <>
       ]
   )
 
+type HTMLexpandable = Interactive ( expanded :: Status )
+
+status :: ∀ r i. Status -> HP.IProp ( expanded :: Status | r ) i
+status = HP.prop (PropName "expanded")
+
+extractStatus
```

Comment:
I sort of remember what this is doing -- pulling the `expanded` label off the row -- but this could use a comment or two explaining what it's doing. Perhaps you @whoadave or @foresttoney or @crcornwell have a short explanation of it worth adding?